app/views/authorised_to_view/authorised_to_view_request_received.scala.html

```diff
@@ -17,20 +17,24 @@
 @import config.AppConfig
 @import uk.gov.hmrc.govukfrontend.views.html.components.FormWithCSRF
 
-@this(
-        layout: Layout,
-        p: components.p,
-        link: components.link,
-        panel: components.confirmation_panel
-)
+@this(layout: Layout,
+      p: components.p,
+      link: components.link,
+      panel: components.confirmation_panel,
+      h2: components.h2)
 
 @(email: String)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
  @layout(
   pageTitle = Some(messages("cf.authorities.request.received.title")),
   helpAndSupport = false
  ) {
-  @panel("cf.authorities.request.received.panel.h1")
-  @p(messages("cf.authorities.request.received.p", email))
-  @link(linkMessage = "cf.authorities.request.received.link", routes.CustomsFinancialsHomeController.index.url)
- }
\ No newline at end of file
+    <div>
+        <a id="cf.authorities.request.received.panel.h1">@panel("cf.authorities.request.received.panel.h1")</a>
+        <a id="cf.authorities.label">@p(messages("cf.authorities.label"))</a>
+        <a id="cf.authorities.next">@h2(messages("cf.authorities.next"))</a>
+        <a id="cf.authorities.next.msg">@p(messages("cf.authorities.next.msg"))</a>
+        <a id="cf.authorities.request.received.link">@link(linkMessage = "cf.authorities.request.received.link",
```

Comment:
Wrapping a link within a link is not valid HTML. Might cause accessibility and usability issues for users.